### ISSUE TITLE
Accessibility: Accessible name for button not descriptive enough

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#52091]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

This buttons name, as read by screen readers, does not describe its purpose.   The button does not  include the type of filters, such as 'Price', 'Numeric Size', 'Size' with their labels. It could be possible for multiple filters to have the same values (for example a width filter and a length filter could both have a value of 12"), if this were the case, users would would not know if they are removing the 12" filter for length or width.

![Image](https://github.com/user-attachments/assets/23e85307-c8d8-423c-ab96-b15b5bdf9a06)

### Component or Page: 

Component: Active Filters Controls Block     

### Name affected element:

Buttons to remove a filter added in the Active Filters Controls block when a filter is selected

### Relevant code snippets:

`<button class="wc-block-active-filters__list-item-remove"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="16" height="16" class="wc-block-components-chip__remove-icon" aria-hidden="true" focusable="false"><path d="M12 13.06l3.712 3.713 1.061-1.06L13.061 12l3.712-3.712-1.06-1.06L12 10.938 8.288 7.227l-1.061 1.06L10.939 12l-3.712 3.712 1.06 1.061L12 13.061z"></path></svg><span class="screen-reader-text">Remove Up to $695.00 filter</span></button>`

### Recommended fix:

Make sure the field has a descriptive, unique, accessible name that accurately describes its purpose. 

This input should be named: ""Remove $filter_name $value filter""

I.e. "Remove Price up to $695.00 filter"
